frontend/src/metabase/lib/formatting/numbers.unit.spec.js

```diff
@@ -0,0 +1,73 @@
+import { formatNumber, numberFormatterForOptions } from "./numbers";
+
+describe("formatNumber", () => {
+  it("should the correct currency format (metabase#34242)", () => {
```

Comment:
Added the issue's reference as suggested by @kamilmielnik [on my previous PR](https://github.com/metabase/metabase/pull/34680#discussion_r1368193794)